integrations/sdks/stainless.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Stainless
+description: Automate your SDK example snippets in the API playground
+---
+
+If you use Mintlify's OpenAPI support for your API reference documentation, add the following to your Stainless config:
+
+```yaml
+openapi:
+  code_samples: mintlify
+```
+
+Configure the [OpenAPI setup](/api-playground/openapi/setup#in-the-repo) in your Mintlify docs. To integrate Stainless, modify the GitHub Action that uploads your OpenAPI spec to Stainless so that it pushes the Stainless-enhanced OpenAPI spec into your docs repo like so:
+
+
+```yaml
+name: Upload OpenAPI spec to Stainless and (Mintlify) docs repo
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  stainless:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Push spec and config to Stainless and outputs documented spec
+        uses: stainless-api/upload-openapi-spec-action@main
+        with:
+          stainless_api_key: ${{ secrets.STAINLESS_API_KEY }}
+          input_path: 'config/acme-openapi.yml'
+          config_path: 'config/acme.stainless.yml'
+          output_path: 'config/acme-openapi.documented.json'
+          project_name: 'acme'
```

Comment:
I know you just copied this from our docs, but I don't love that we use acme as a placeholder here. Can we replace with:
```
input_path: 'path/to/my-company-openapi.json'
config_path: 'path/to/my-company.stainless.yaml'
output_path: 'path/to/my-company-openapi.documented.json'
project_name: 'my-stainless-project'
```
to make it a little more blatant that we are using placeholders.

Reply:
@meorphis added